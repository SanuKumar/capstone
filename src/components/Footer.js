import React from 'react'
import { Row, Col } from "react-bootstrap"

const Footer = () => {
  return (
    <footer className='footer'>
      <p style={{ color: "white", textAlign: "center" }}>Sanu Â©. All rights reserved.</p>
    </footer>
  )
}

export default Footer