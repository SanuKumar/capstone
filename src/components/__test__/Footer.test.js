import React from "react"
import { render, cleanup } from "@testing-library/react"
import Footer from "../Footer"

afterEach(cleanup)

describe("Footer page snapshot", () => {
  test("renders correctly", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  })
})