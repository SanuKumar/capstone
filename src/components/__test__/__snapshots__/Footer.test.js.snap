// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer page snapshot renders correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <p
      style="color: white; text-align: center;"
    >
      Sanu Â©. All rights reserved.
    </p>
  </footer>
</DocumentFragment>
`;
